x-pack/test/performance/tests/playwright/login.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { FtrProviderContext } from '../../ftr_provider_context';
+import { StepCtx } from '../../services/performance';
+
+export default function ecommerceDashboard({ getService }: FtrProviderContext) {
+  describe('login', () => {
+    it('login', async () => {
+      const inputDelays = getService('inputDelays');
+      const performance = getService('performance');
+
+      await performance.runUserJourney(
+        'login',
+        [
+          {
+            name: 'Login',
+            handler: async ({ page, kibanaUrl }: StepCtx) => {
+              await page.goto(`${kibanaUrl}`);
+
+              const usernameLocator = page.locator('[data-test-subj=loginUsername]');
+              const passwordLocator = page.locator('[data-test-subj=loginPassword]');
+              const submitButtonLocator = page.locator('[data-test-subj=loginSubmit]');
```

Comment:
Shouldn't we stop the `Login` journey when the page is rendered?

Reply:
Let's discuss that and then I can create a separate PR if we decide to do that.